package.json

```diff
@@ -23,7 +23,7 @@
     "beepbeep": "^1.2.0",
     "browser-sync": "^2.11.0",
     "colors": "^1.1.2",
-    "gulp": "git+https://github.com/gulpjs/gulp.git#4.0",
+    "gulp": "git+https://github.com/gulpjs/gulp.git#v4.0.0",
```

Comment:
This does not really solve the issue. the `#4.0` was a reference to a branch, not a tag. The 4.0 branch is deleted a a tag is released for this. If you change it to `"gulp": "^4.0.0",` it would the correct way.

Reply:
`#4.0.0` seems to now be reference a non-existing branch so it's broken again. What is specific about needing to reference a particular branch here rather than a semver?

Reply:
For a long time there was no version 4 available, only the branch. In the meanwhile there is a tagged version for 4 and the branch was deleted. 

Which mean that's wrong in the template und should be changed to `"gulp": "^4.0.0"`. There are other PRs (#56) which should fix this, but unfortunately nobody does merged them.